static/js/util/profile_edit.js

```diff
@@ -220,6 +221,57 @@ export function boundCheckbox(keySet: string[], label: string|React$Element<*>):
   );
 }
 
+const onTelChange = R.curry((
+  keySet,
+  countryKeySet,
+  profile,
+  updateProfile,
+  validator,
+  newPhoneNumber,
+  countryObj
+) => {
+  let currentPhoneNumber = _.get(profile, keySet);
+  if ( currentPhoneNumber !== newPhoneNumber ) {
+    let clone = _.cloneDeep(profile);
+    _.set(clone, keySet, newPhoneNumber);
+    _.set(clone, countryKeySet, countryObj.iso2);
```

Comment:
do you think we still need to save that if the country code is already in the phone number field? when the user types a number into the input it will format it automatically with a country code.